schemas/oic.r.session-request.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema" : "http://json-schema.org/draft-04/schema#",
+  "description" : "Copyright (c) 2017 Open Connectivity Foundation, Inc. All rights reserved.",
+  "title" : "Account Session",
+  "id": "https://www.openconnectivity.org/ocf-apis/security/schemas/oic.r.session-request.json#",
+  "definitions" : {
+    "uuid" : {
+      "type" : "string",
+      "pattern" : "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+    }
+  },
+  "type" : "object",
+  "properties" : {
+    "uid" : {
+      "allOf": [
```

Comment:
Using "allof" in this situation might be causing schema validation issues with our tooling.  Maybe Mark can suggest what this should look like.

Reply:
Done.